english/watch-movie/Up.md

```diff
@@ -1,4 +1,4 @@
 You decided to asked the marshmallow to watch your favourite movie Up with you.
 Halfway through the movie, the marshmallow walks out.  
-You change the TV to F-Troop and it comes back in and sits down.  
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.